### ISSUE TITLE
epic: Compute type Management

### ISSUE THREAD

**Issue by davidfestal** [kcp-dev/kcp#1895]
### TL;DR

Centrally manage the standard Compute API resources used in SyncTargets

Mostly what is sketched in [this drawing](https://docs.google.com/drawings/d/1aiX_1xc-dpvYJZBXVhD8g0maKFNFT-M3M31DjFWsZSo/edit)

### Progress tracking <a href="#progress-tracking" id="progress-tracking">#</a>

EPIC detailed issues and overall progress can be tracked with the following [project view](https://github.com/orgs/kcp-dev/projects/1/views/32)

### Demo Objective

- [ ] User should be able to refer to global compute Workspaces, to bind standard already-available APIExports for standard compute resources like Kubernetes, Openshift, KubeVirt, etc ...
- [ ] The source of truth for the schemas of the Standard compute APIs should not anymore come through importing them from the SyncTarget.
- [ ] The Syncer should still import the corresponding resource schemas to KCP, but only to detect incompatible APIs between the global compate APIs and the APIs provided by the SyncTarget, and drive workload scheduling accordingly.
- [ ] There should be some sort of way to publish / maintain / version those global standard compute APIExports and related schemas from the code.
- [ ] The user should have a view of the locations he has access to, along with the compute APIExports it serves.

### Demo Steps

1. Admin creates a SyncTarget in a location workspace referencing the `kubernetes` root compute APIExport
2. Admin creates a Location in the location workspace that includes this Synctarget
3. User binds to the `kubernetes` root compute APIExport in his user workspace
4. User list the locations he has access to and find the location created by admin in step 2
5. User creates a placement for this location
6. User creates a Deployment
7. Deployment is synced to the SyncTarget (unless the Deployment schema in the Synctarget is not compatible with the Deployment schema in the `kubernetes` root APIExport.

### Action Items

- [X] Scope of the current demo is necessary to fit in the prototype boundaries
- [ ] Contribute to the final demo script and recording

### Stories

- [x] https://github.com/kcp-dev/kcp/issues/1959
- [x] https://github.com/kcp-dev/kcp/issues/1888
- [x] #1960
- [x] #2102
- [x] https://github.com/kcp-dev/kcp/issues/1779

==== stretch goals

- [ ] Provide the user a way to see the locations which are available for him, with the corresponding labels, as well as the referenced APIExports
- [ ] Disable the automatic creation of APIResourceschemas and completion of the `kubernetes` APIExport based on `NegotiatedAPIResource`s (workload apiexport controller) - Well, do we really want to remove that, or might we want to rename the local `kubernetes` APIExport to something else, ie `imported`, and keep the ability to import individual resources directly from SyncTargets ?

**Comment by davidfestal** [comment]
cc @sttts @qiujian16

**Comment by davidfestal** [comment]
issue https://github.com/kcp-dev/kcp/issues/2349 should be fixed before 0.10.0 (followup bug).
A PR is already in review.

**Comment by ncdc** [comment]
Does fixing that finish this epic?

**Comment by davidfestal** [comment]
Yes, we can say so.
